runing/models.py

```
@@ -0,0 +1,10 @@
+from sqlalchemy import Column, Integer, String, Boolean
+from database import Base
+class User(Base):
+    __tablename__ = "users"
+
+    id = Column(Integer, primary_key =True, index =True)
+    email = Column(String, unigue=true, index=True)
```

Ruff (0.11.9)

Undefined name `true`

(F821)